### PR TITLE
Fixed parsing bugs

### PR DESCRIPTION
**Pull request by marsaud**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -25,11 +25,11 @@ export const parse = url => {
   } else if (type === 'smb') {
     remote.type = 'smb'
     const lastArobas = rest.lastIndexOf('@')
```
Comment (julien-f) [inline src/index.js]: The correct English word is [_at sign_](https://en.wikipedia.org/wiki/At_sign).

File: src/index.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ export const parse = url => {
     remote.username = username
     remote.password = password
   }
+  remote.url = url
```
Comment (julien-f) [inline src/index.js]: Can be moved directly in `remote` declaration.

File: src/index.spec.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ const data = {
     url: 'smb://Administrator:password@toto\\\\192.168.100.225\\smb\0',
     type: 'smb',
     host: '192.168.100.225\\smb',
-    path: undefined,
+    path: '',
```
Comment (julien-f) [inline src/index.spec.js:22]: Does it make sense in SMB to have an empty path?

Comment (marsaud) [inline src/index.spec.js:22]: Smb Handler tests against path === ''

https://github.com/vatesfr/xo-server/blob/next-release/src/remote-handlers/smb.js#L53

Comment (julien-f) [inline src/index.spec.js:22]: I see.

**Pull request merged**